src/Common/LoggingConstants.cs

```diff
@@ -13,6 +13,9 @@ public static class LoggingConstants
         public const string ApplicationInsightsInstrumentationKeyEnvironmentVariableName
             = "ORYX_AI_INSTRUMENTATION_KEY";
 
+        public const string DisableOryxTelemetryEnvironmentVariableName
+            = "DISABLE_ORYX_TELEMETRY";
```

Comment:
our environment variables usually start with 'Oryx', which makes it easy to find all variables related to us. So I'd call it 'ORYX_DISABLE_TELEMETRY' instead.